packages/core/src/utils.test.ts

```diff
@@ -237,7 +237,10 @@ describe('Core Utils', () => {
   });
 
   test('Capitalize', () => {
+    expect(capitalize('id')).toEqual('ID');
+    expect(capitalize('Id')).toEqual('ID');
     expect(capitalize('foo')).toEqual('Foo');
+    expect(capitalize('FOO')).toEqual('FOO');
```

Comment:
Test non-alpha strings to make sure nothing breaks

Reply:
https://github.com/medplum/medplum/pull/408/commits/97d82f5a1660d57a7b2b9929a0e54075ba871f3c